go.mod

```diff
@@ -1,20 +1,23 @@
 module github.com/openshift/configure-alertmanager-operator
 
-go 1.17
+go 1.19
 
 require (
-	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/logr v1.2.4
 	github.com/golang/mock v1.5.0
-	github.com/openshift/api v0.0.0-00010101000000-000000000000
+	github.com/onsi/ginkgo/v2 v2.9.7
+	github.com/onsi/gomega v1.27.7
+	github.com/openshift/api v0.0.0-20220414050251-a83e6f8f1d50
+	github.com/openshift/osde2e-common v0.0.0-20230607130803-f152ba7eaf6e
 	github.com/operator-framework/operator-lib v0.11.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.56.3
-	github.com/prometheus/client_golang v1.12.1
-	github.com/prometheus/common v0.32.1
+	github.com/prometheus/client_golang v1.15.1
+	github.com/prometheus/common v0.43.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.24.3
-	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v0.24.2
-	sigs.k8s.io/controller-runtime v0.12.3
+	k8s.io/api v0.27.2
+	k8s.io/apimachinery v0.27.2
+	k8s.io/client-go v0.27.2
+	sigs.k8s.io/controller-runtime v0.15.0
```

Comment:
These aren't even going to have an effect because they are replaced at the bottom

Reply:
they are bumped when we run go mod tidy because of the dependencies, it does not matter if they are replaced. Things work as intended within the tests regardless of the pinning below.